noop/unikernel.ml

```diff
@@ -0,0 +1,6 @@
+module Main = struct
```

Comment:
If this is the first example someone sees in a tutorial, making the structure different between this and the next thing (which is likely to be something that requires a `Main`, since it's a functor) needs an explanation in the next thing.  I'm not sure whether that's bad or not, though.